tests/test_checkpoint.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from testify import *
+import tempfile
+import sqlite3
+import mock
+import logging
+import sys
+
+from triton import checkpoint
+
+level = logging.DEBUG
+log_format = "%(asctime)s %(levelname)s:%(name)s: %(message)s"
+logging.basicConfig(level=level, format=log_format)
+
+log = logging.getLogger(__name__)
+
+
+CREATE_TABLE_STMT = """
+CREATE TABLE IF NOT EXISTS triton_checkpoint (
+    client VARCHAR(255) NOT NULL,
+    stream VARCHAR(255) NOT NULL,
+    shard VARCHAR(255) NOT NULL,
+    seq_num VARCHAR(255) NOT NULL,
+    updated INTEGER NOT NULL,
```

Comment:
Currently, only `seq_num` and `updated` have a not null constraint.  Is this disparity intentional?